src/ui/components/search/autocompletecomponent.js

```diff
@@ -358,6 +374,23 @@ export default class AutoCompleteComponent extends Component {
     this._isVoiceSearchActive = value;
   }
 
+  setCustomPrompts (prompts) {
+    this.customPrompts = prompts;
+  }
+
+  combineResultsWithCustomPrompts (data) {
+    const newSections = data.sections.filter(result => result.resultsCount !== 0);
```

Comment:
Small nit: Could we break this method up? Maybe have a `createSectionFromCustomPrompts` and then the combine method?